DayrSuryan/DSEthiop12.xml

```diff
@@ -119,7 +119,9 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         አርዳኢሁ፡ ድኅረ፡  ትንሳ<del rend="erasure" unit="chars" extent="1"/>ኤሁ፡
                            ቅድመ፡ <hi rend="rubric">እርገቱ፡ በስጋ፡</hi> ለመድ<add place="above">ሀ</add>ኒነ፡
                            ዘመላ፡ ሰማያተ፡ ወምድረ፡ ቅድሳተ፡ ስብሃቲሁ፡<hi rend="rubric">ጸሎቱ፡ ወበረ<supplied reason="omitted" resp="DR">ከ</supplied>ቱ፡
-                           የሃሉ፡ ምስለ፡</hi>
+                           የሃሉ፡ ምስለ፡<gap reason="omitted" resp="DN"/>
```

Comment:
Is it possible that there is a problem here with the opening and closing of hi? Could you check again in the manuscript?

Reply:
done